### DISCUSSION TITLE
How to change font size in chakra ui?

### DISCUSSION THREAD

**Discussion by ameeetgaikwad** [ethereum/ethereum-org-website#7783] [Q&A] [answered]
Do I have to use xs, sm, md, lg, etc to define the size or can I use numbers like 1.25?

I have to refactor this code:
```typescript
   const Title = styled.p`
   font-size: 1.25rem;
 `
```

So should I go like this? 
```typescript
<Text as="p" fontSize="xl" />
```
 or this 
```typescript
<Text as="p" fontSize="1.25" />
```

**Comment by MahendraBishnoi29** [comment, marked as answer]
hey @ameeetgaikwad you should use `xl`

<img width="146" alt="Screenshot 2022-09-10 133523" src="https://user-images.githubusercontent.com/74294202/189474903-96ade160-e415-4a1f-9ef7-73996de7e5fc.png">

**Reply by ameeetgaikwad** [reply]
Thank you mate!

**Reply by MahendraBishnoi29** [reply]
np bro :)

**Reply by ameeetgaikwad** [reply]
BTW where did you get this ss from?

**Reply by MahendraBishnoi29** [reply]
official chakra UI typography docs 🙂

**Reply by pettinarip** [reply]
Cool yes! as @MahendraBishnoi29 said. Check all the possible values in [this page of the Chakra docs](https://chakra-ui.com/docs/styled-system/theme#typography).

**Reply by Packmichael** [reply]
@ericfx_crypto_trader on in-sta-gram has help me a lot through his good strategies,I’m earning more than a thousand pounds daily with his skills +❶ (❼❽❻) ❽❹❶‑❺⓿❹❺
